ci/ray_ci/macos/macos_ci.sh

```diff
@@ -11,6 +11,8 @@ export LANG="en_US.UTF-8"
 export BUILD="1"
 export DL="1"
 export RUN_PER_FLAKY_TEST="2"
+export TORCH_VERSION=2.0.1
```

Comment:
this is the only delta compared to https://github.com/ray-project/ray/commit/13d02a3c198279ceffcd9e40a516f854dd714f73 to NOT upgrading torch versions for macos